lib/katello/tasks/pulp3_content_switchover.rake

```diff
@@ -1,3 +1,5 @@
+load "#{Katello::Engine.root}/lib/katello/tasks/common.rake"
+
 require File.expand_path("../engine", File.dirname(__FILE__))
```

Comment:
Unrelated, but I wonder why this isn't:
```suggestion
require File.expand_path("engine", File.dirname(__dir__))
```

Reply:
i'm not sure this line is even needed, but lets go ahead and merge this